### PR TITLE
MM-27719: use multitenant rds as datastore and s3 as filestore

### PR DESCRIPTION
**Pull request by cpanato**
#### Summary
Set the database to use multitenant and s3 as filestore

#### Ticket Link
Jira: https://mattermost.atlassian.net/browse/MM-27719

#### Release Note

```release-note
spinwick: use multitenant rds as datastore and s3 as filestore
```

### CODE REVIEW

File: server/spinwick.go [hunk 1]
```diff
@@ -290,12 +290,14 @@ func (s *Server) createSpinWick(pr *model.PullRequest, size string, withLicense
...
-		DNS:      fmt.Sprintf("%s.%s", ownerID, s.Config.DNSNameTestServer),
-		Size:     size,
-		Affinity: "multitenant",
+		OwnerID:   ownerID,
+		Version:   version,
+		Image:     image,
+		DNS:       fmt.Sprintf("%s.%s", ownerID, s.Config.DNSNameTestServer),
+		Size:      size,
+		Affinity:  "multitenant",
+		Database:  "aws-multitenant-rds",
```
Comment (gabrieljackson) [inline server/spinwick.go:299]: The postgres multitenant value is actually different: https://github.com/mattermost/mattermost-cloud/blob/a981c07c93e5a49d26710623eaa9eb0b788bbd37/model/installation_database.go#L30

I would make sure the mattermost-cloud repo dependancy is up to date and use `cloudModel.InstallationDatabaseMultiTenantRDSPostgres` instead.

Comment (cpanato) [reply server/spinwick.go:299]: right! thanks for the catch!

Comment (cpanato) [reply server/spinwick.go]: done @gabrieljackson

**Comment by cpanato** [comment]
also did a cleanup

**Pull request merged**